eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -388,7 +388,7 @@ stages:
                   - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
                     - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
               WorkingDirectory: $(System.DefaultWorkingDirectory)
-              TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+              TargetDocRepoOwner: "azure-sdk"
```

Comment:
Should we update the other location as well? Can the TargetDocRepoOwner parameter be removed now?

Reply:
The other one is to update Azure repo main branch. We can hard-coded like this as well as it only used once in yaml.

Reply:
I think hardcoding is the way to go here and we can remove the parameter from `archetype-java-release.yml` and the pipelines calling it. 

`archetype-java-release.yml` gets used in: 
* https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/cosmos-sdk-client.yml
* https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/stages/archetype-sdk-client.yml